app/src/main/kotlin/com/softteco/template/utils/AppDispatchers.kt

```diff
@@ -0,0 +1,5 @@
+package com.softteco.template.utils
+
+import kotlinx.coroutines.CoroutineDispatcher
+
+data class AppDispatchers(val io: CoroutineDispatcher)
```

Comment:
Would be good to have all possible dispatchers here: 

```
data class AppDispatchers(
    val ui: CoroutineDispatcher,
    val default: CoroutineDispatcher,
    val io: CoroutineDispatcher,
    val unconfined: CoroutineDispatcher
)

```

Reply:
@anastasiaShpakava could you also use this logic in the whole app, if somewhere Dispatchers is hardcoded - then change it to the dispatcher from AppDispatchers